pages/api/getLocations/[uuid].js

```diff
@@ -13,13 +13,16 @@ export default async (req, res) => {
 				result = await pool.query(`
 					SELECT  locations.school_id,
 							locations.lat,
-							locations.lon
+							locations.lon,
+							countries.country_name,
```

Comment:
Following the argument above, you don't need to retrieve `country_name`; you can pull it from the code directly when it needs to be displayed to the user.